### PR TITLE
[cutelyst2] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469  
Depends on #21568

### CODE REVIEW

File: ports/cutelyst2/vcpkg.json [hunk 1]
```diff
@@ -1,12 +1,20 @@
 {
   "name": "cutelyst2",
   "version-string": "2.12.0",
```
Comment (JonLiu1993) [inline ports/cutelyst2/vcpkg.json:3]:
```suggestion
  "version": "2.12.0",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@autoantwort ,Thanks for your pr, could you please take a look:
````
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:78 (message):
  D:/downloads/tools/msys2/aa5af7b2aa7e90e8/mingw32/bin/pkg-config.exe
  --exists Cutelyst2Qt5Authentication failed with error code: 1

      ENV{PKG_CONFIG_PATH}: "D:/packages/cutelyst2_x86-windows/lib/pkgconfig;D:/packages/cutelyst2_x86-windows/share/pkgconfig;D:/installed/x86-windows/lib/pkgconfig;D:/installed/x86-windows/share/pkgconfig"
      output: Package Qt5Core was not found in the pkg-config search path.

  Perhaps you should add the directory containing `Qt5Core.pc'

  to the PKG_CONFIG_PATH environment variable

  Package 'Qt5Core', required by 'Cutelyst2Qt5Authentication', not found
````

**Comment by autoantwort** [comment]
Now depends on #21695

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the many absolute paths fixes and for modernizing this port!

**Pull request merged**